app/tasks/maintenance.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+global $cli;
+
+require_once __DIR__.'/../init.php';
+
+use Appwrite\Database\Database;
+use Appwrite\Database\Document;
+use Appwrite\Database\Adapter\MySQL as MySQLAdapter;
+use Appwrite\Database\Adapter\Redis as RedisAdapter;
+use Appwrite\Database\Validator\Authorization;
+use Utopia\App;
+use Utopia\CLI\Console;
+use Utopia\Config\Config;
+
+define("DELETE_QUEUE_NAME", "v1-deletes");
```

Comment:
For consistency, let's add new constants for all the queues, better add them in our init.php constants list if no relevant class is available. Actually maybe we can add them in the Event class. Let me know what you think.

When setting new constants, we also better be consistent and use the same syntax. Right now we have preferred the `const X = 'x';` over `define('X', 'x');`. +Single quotes have better performance than double quotes in PHP.

Reply:
Thanks eldad. Those are some useful insights! I will add them in Events. That is more relevant